### PR TITLE
Various Linting Fix

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR adds a `doc.go` file for each package to describe the package content to comply with Linting rules.

### CODE REVIEW

File: internal/auth-service/auth-service-config.go [hunk 1]
```diff
@@ -3,17 +3,18 @@ package auth_service
 import (
 	"reflect"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline internal/auth-service/auth-service-config.go:6]: Should be grouped together with the other

File: internal/auth-service/auth_test.go [hunk 2]
```diff
@@ -15,6 +15,8 @@ import (
 	"testing"
 	"time"
 
+	testUtils2 "github.com/liqotech/liqo/pkg/utils/testUtils"
```
Comment (giorio94) [inline internal/auth-service/auth_test.go:18]: Same

**Review by alacuku — APPROVED** [review]

File: apis/net/v1alpha1/networkconfig_types.go [hunk 3]
```diff
@@ -31,7 +31,7 @@ type NetworkConfigSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// The ID of the remote cluster that will receive this CRD.
-	ClusterID string `json:"clusterID"`
+	ClusterID string `json:"clusterid"`
```
Comment (giorio94) [inline apis/net/v1alpha1/networkconfig_types.go:34]: Is this breaking change desired?

File: apis/net/v1alpha1/tunnel_endpoint_types.go [hunk 4]
```diff
@@ -27,7 +27,7 @@ type TunnelEndpointSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 	// The ID of the remote cluster that will receive this CRD.
-	ClusterID string `json:"clusterID"`
+	ClusterID string `json:"clusterid"`
```
Comment (giorio94) [inline apis/net/v1alpha1/tunnel_endpoint_types.go:30]: Same here

File: internal/discovery/foreign-cluster-operator/foreign-cluster-controller.go [hunk 5]
```diff
@@ -23,13 +23,14 @@ import (
 	"strings"
 	"time"
 
+	"github.com/liqotech/liqo/pkg/clusterid"
+
```
Comment (giorio94) [inline internal/discovery/foreign-cluster-operator/foreign-cluster-controller.go:27]: The empty line should be removed

File: internal/crdReplicator/crdReplicator-config.go [hunk 6]
```diff
@@ -3,13 +3,14 @@ package crdReplicator
 import (
 	"reflect"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline internal/crdReplicator/crdReplicator-config.go:6]: Same here

File: internal/discovery/discovery-config.go [hunk 7]
```diff
@@ -4,13 +4,14 @@ import (
 	"context"
 	"reflect"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline internal/discovery/discovery-config.go:7]: Same here

File: internal/discovery/foreign-cluster-operator/foreign-cluster-operator_test.go [hunk 8]
```diff
@@ -5,6 +5,8 @@ import (
 	"path/filepath"
 	"testing"
 
+	testUtils2 "github.com/liqotech/liqo/pkg/utils/testUtils"
```
Comment (giorio94) [inline internal/discovery/foreign-cluster-operator/foreign-cluster-operator_test.go:8]: Same here

File: internal/discovery/foreign-cluster-operator/setup.go [hunk 9]
```diff
@@ -9,11 +9,12 @@ import (
 	"k8s.io/klog"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
+	"github.com/liqotech/liqo/pkg/clusterid"
```
Comment (giorio94) [inline internal/discovery/foreign-cluster-operator/setup.go:12]: Same here

File: internal/discovery/search-domain-operator/wan_test.go [hunk 10]
```diff
@@ -3,11 +3,12 @@ package search_domain_operator
 import (
 	"testing"
 
+	testUtils2 "github.com/liqotech/liqo/pkg/utils/testUtils"
```
Comment (giorio94) [inline internal/discovery/search-domain-operator/wan_test.go:6]: Same here

File: internal/liqonet/tunnel-operator/config.go [hunk 11]
```diff
@@ -3,13 +3,14 @@ package tunnel_operator
 import (
 	"os"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline internal/liqonet/tunnel-operator/config.go:6]: Same here

File: internal/liqonet/tunnelEndpointCreator/tunnelEndpointCreator-config.go [hunk 12]
```diff
@@ -4,14 +4,15 @@ import (
 	"os"
 	"reflect"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline internal/liqonet/tunnelEndpointCreator/tunnelEndpointCreator-config.go:7]: Same here

File: internal/liqonet/tunnelEndpointCreator/tunnelEndpointCreator-operator.go [hunk 13]
```diff
@@ -25,6 +25,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline internal/liqonet/tunnelEndpointCreator/tunnelEndpointCreator-operator.go:28]: Same here

File: internal/peering-request-operator/peering-request-controller.go [hunk 14]
```diff
@@ -21,6 +21,8 @@ import (
 	"errors"
 	"time"
 
+	"github.com/liqotech/liqo/pkg/clusterid"
```
Comment (giorio94) [inline internal/peering-request-operator/peering-request-controller.go:24]: Same here

File: pkg/auth/identityResponse.go [hunk 15]
```diff
@@ -4,11 +4,12 @@ import (
 	"encoding/base64"
 	"io/ioutil"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline pkg/auth/identityResponse.go:7]: Same here

File: pkg/clusterid/const.go [hunk 16]
```diff
@@ -0,0 +1,7 @@
+package clusterid
+
+const (
+	masterLabel    = "node-role.kubernetes.io/master"
+	serviceAccount = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
```
Comment (giorio94) [inline pkg/clusterid/const.go:5]: Is it correct those are in the clusterid package?

File: pkg/identityManager/identityManager_test.go [hunk 17]
```diff
@@ -9,17 +9,18 @@ import (
 	"testing"
 	"time"
 
+	testUtils2 "github.com/liqotech/liqo/pkg/utils/testUtils"
```
Comment (giorio94) [inline pkg/identityManager/identityManager_test.go:12]: Should be merged

File: pkg/identityManager/signingRequest.go [hunk 18]
```diff
@@ -8,6 +8,8 @@ import (
 	"strings"
 	"time"
 
+	certificateSigningRequest2 "github.com/liqotech/liqo/pkg/utils/certificateSigningRequest"
```
Comment (giorio94) [inline pkg/identityManager/signingRequest.go:11]: Should be merged and the 2 removed

File: pkg/kubeconfig/create.go [hunk 19]
```diff
@@ -5,6 +5,8 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/liqotech/liqo/pkg/utils"
```
Comment (giorio94) [inline pkg/kubeconfig/create.go:8]: Should be merged

File: pkg/tenantControlNamespace/tenantControlNamespaceManager_test.go [hunk 20]
```diff
@@ -8,6 +8,8 @@ import (
 	"strings"
 	"testing"
 
+	testUtils2 "github.com/liqotech/liqo/pkg/utils/testUtils"
```
Comment (giorio94) [inline pkg/tenantControlNamespace/tenantControlNamespaceManager_test.go:11]: Should be merged

**Review by giorio94 — APPROVED** [review]
/lgtm

Some minor comments inline

**Pull request merged**